### ISSUE TITLE
Option to turn off Obfuscate debug reporting

### ISSUE THREAD

**Issue by watercrossing** [phw198/OutlookGoogleCalendarSync#1211]
**OGCS Version**: v2.8.6.0
**Installed or Portable**: Installed

**Problem Description**  
I obfuscate every calendar item when synchronising Google -> Outlook. The rule is  `^.*$` -> `PrivateEvent`.

This works fine, but the log is being spammed by non-useful DEBUG messages:

```
2021-04-17 09:14:09,372 INFO  11 OutlookGoogleCalendarSync.Console [282] -  Outlook 244, Google 52
2021-04-17 09:14:09,466 DEBUG  11 OutlookGoogleCalendarSync.Console [280] -  Checking for orphaned Outlook items...
2021-04-17 09:14:09,466 DEBUG  11 OutlookGoogleCalendarSync.OutlookOgcs.Calendar [740] -  Scanning 244 Outlook appointments for orphans to reclaim...
2021-04-17 09:14:09,544 DEBUG  11 OutlookGoogleCalendarSync.Obfuscate [87] -  Regex has matched and altered string: ^.*$
... repeat 10000 times (pretty much exactly 10000 times)
2021-04-17 09:14:24,325 DEBUG  11 OutlookGoogleCalendarSync.Obfuscate [87] -  Regex has matched and altered string: ^.*$
```

This doesn't seem neither useful or sensible: 
- The debug messages don't seem to help with debugging
- There are now 10s of debug log files per day (since they are limited to 2MB)

**Steps to Reproduce the Issue**  
1. Set obfuscation rule to  `^.*$` -> `PrivateEvent`.
2. Set loglevel to DEBUG or lower.

**Comment by phw198** [comment]
Yep, not particularly useful!

:package: This should now be resolved with [hotfix v2.8.6.11.zip](https://github.com/phw198/OutlookGoogleCalendarSync/files/6412141/v2.8.6.11.zip) ([see instructions for applying](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-Problems#applying-a-hotfix)).

**Comment by watercrossing** [comment]
Thank you, the hotfix fixed it.
